src/content/en/progressive-web-apps/_index.yaml

```diff
@@ -93,9 +93,9 @@ landing_page:
         <p>
           The Web App Manifest allows you to control how your app appears and
           how it's launched. You can specify icons for the home screen and
-          splash screen which is shown while the app is loading. Which page
-          is loaded when the app is launched, screen orientation, even
-          whether to show the browser chrome or not.
+          the splash screen shown while the app loads, which page to load when
+          the app launches, screen orientation, even whether or not to show the
+          browser chrome.
         </p>
```

Comment:
I interpreted the original text differently: It seems like [the `icons` property is used for the home screen and the loading screen](https://developers.google.com/web/updates/2015/10/splashscreen), as opposed to being a separate properties allowing specific tweaking of the "splash screen". TBH it seems like a minor and confusing point (especially if we're advertising "load instantly") , so here's how it flows without mentioning the splash screen.